src/node_crypto.cc

```diff
@@ -5424,8 +5424,11 @@ void PBKDF2(const FunctionCallbackInfo<Value>& args) {
   if (args[5]->IsFunction()) {
     obj->Set(env->ondone_string(), args[5]);
 
-    if (env->in_domain())
-      obj->Set(env->domain_string(), env->domain_array()->Get(0));
+    if (env->in_domain()) {
+      obj->Set(env->domain_string(),
+               env->domain_array()->Get(env->context(), 0).ToLocalChecked());
```

Comment:
Any reason not to call `.FromJust()` here and below as well (à la `RandomBytesBuffer()`)?